### ISSUE TITLE
Code Table Request - Add Accession as ID type

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6048]
**How to Use This Form**
This is a template with examples and guidance on how to best communicate with the Arctos Working Group. Please delete this section along with anything in square brackets ```[ ] ``` below before submitting. 

[ **Instructions for reference:** ]
[ Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html ]

[ Code Table Documentation is https://handbook.arctosdb.org/how_to/How-to-Use-Code-Tables.html ]

[[Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)]

**Goal**
Add accession as a value to the Other ID code table to be able to accommodate identifiers from external institutions, such as National Park Service Accession (or accession numbers assigned by specific national parks.)

**Context**
Accession does not currently exist as an other ID.

**Table**
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type

Proposed Value**
accession number

**Proposed Definition**
Accessioning is the process of officially accepting items into National Park
Service (NPS) and other museum collections. Accessioning establishes legal
custody and ownership and provides information on how the museum acquired
the items.  For NPS, the accession number consists of the park
acronym, a dash, and a number.
https://www.nps.gov/museum/publications/mhii/mh2ch2firstfile.pdf
**Available for Public View**
Yes

**Other ID BaseURL**
[  For OtherIDs, provide the following *or* explain why the unresolvable ID type is necessary:
1. "Base URL" with which to prepend entered values, and
2. A functional example URL, which should consist of the base URL provided in (1) plus a relevant value.  ]

**ID_References**
[  If the request involves https://arctos.database.museum/info/ctDocumentation.cfm?table=ctid_references, the changes must be coordinated with the Database Admin team for notifications to function.  ]

**Priority**
Medium. Related to #5868 and #6012

**Comment by campmlc** [comment]
I agree with the editing, but omitting the "often". Accessioning in the museum curatorial sense establishes ownership. I agree with just using accession, without the number. Checking the code table box . . .

**Comment by campmlc** [comment]
Updated proposed value and definition. Checked code table box. Waiting on second Code Table admin approval.

**Comment by dustymc** [comment]
Is everyone OK with replacing https://github.com/ArctosDB/arctos/issues/5968 with this? Is there some other use case? I'm tempted to close both and revisit when we've got through some things that might be less sticky.

**Comment by Jegelewicz** [comment]
I am happy to sign off on this once I am certain that it isn't going to somehow cause issues by being the same term as this:

![image](https://user-images.githubusercontent.com/5725767/229571718-8592ad55-2a8f-42cc-b1b4-f5c8f7c4cd95.png)

See also https://github.com/ArctosDB/arctos/issues/5448

**Comment by campmlc** [comment]
For that reason, perhaps we need accession number?

**Comment by Jegelewicz** [comment]
I found another use case. See https://github.com/ArctosDB/arctos/issues/6350

**Comment by Jegelewicz** [comment]
See also #6436

**Comment by Jegelewicz** [comment]
@mkoo this needs a second code table admin sign off, but I am still wondering about https://github.com/ArctosDB/arctos/issues/6048#issuecomment-1494636976

**Comment by dustymc** [comment]
I vote to table this, https://github.com/ArctosDB/arctos/issues/5969, and https://github.com/ArctosDB/arctos/issues/5968, just because I don't think these are causing issues, some other ID types clearly are, and our resources are stretched thin. 

Otherwise we MUST have a path to merging when this is created; we just can't create yet another arbitrary pathway.

**Comment by Jegelewicz** [comment]
I'd prefer NOT to add this and favor the use of appropriate issued by.

[United States National Park Service Catalog](https://arctos.database.museum/agent/21347816)
[United States National Park Service Register](https://arctos.database.museum/agent/21347817)

**Comment by dustymc** [comment]
>favor the use of appropriate issued by

Yes, that's the normalized/correct way to handle this, I'm just suggesting that perhaps we should focus more on the things that are actively causing problems. (And maybe this situation is and I just don't see it, it's hard to see much of anything from the weird limbo we seem to be stuck in.)
